openslides/users/urls.py

```diff
@@ -4,75 +4,10 @@
 
 urlpatterns = patterns(
     '',
-    # User
-    url(r'^$',
-        views.UserListView.as_view(),
-        name='user_list'),
-
-    url(r'^new/$',
-        views.UserCreateView.as_view(),
-        name='user_create'),
-
-    url(r'^new_multiple/$',
-        views.UserMultipleCreateView.as_view(),
-        name='user_create_multiple'),
-
-    url(r'^(?P<pk>\d+)/$',
-        views.UserDetailView.as_view(),
-        name='user_detail'),
-
-    url(r'^(?P<pk>\d+)/edit/$',
-        views.UserUpdateView.as_view(),
-        name='user_update'),
-
-    url(r'^(?P<pk>\d+)/del/$',
-        views.UserDeleteView.as_view(),
-        name='user_delete'),
-
-    url(r'^(?P<pk>\d+)/reset_password/$',
-        views.ResetPasswordView.as_view(),
-        name='user_reset_password'),
-
-    url(r'^(?P<pk>\d+)/status/activate/$',
-        views.SetUserStatusView.as_view(),
-        {'action': 'activate'},
-        name='user_status_activate'),
-
-    url(r'^(?P<pk>\d+)/status/deactivate/$',
-        views.SetUserStatusView.as_view(),
-        {'action': 'deactivate'},
-        name='user_status_deactivate'),
-
-    url(r'^(?P<pk>\d+)/status/toggle/$',
-        views.SetUserStatusView.as_view(),
-        {'action': 'toggle'},
-        name='user_status_toggle'),
-
     url(r'^csv_import/$',
         views.UserCSVImportView.as_view(),
```

Comment:
I thought we discussed that @emanuelschuetze first creates all apps via angular before he create the csv stuff.